packages/turf-clone/index.js

```diff
@@ -70,12 +77,21 @@ function cloneFeature(feature) {
  * @returns {FeatureCollection<any>} cloned Feature Collection
  */
 function cloneFeatureCollection(geojson) {
-    return {
-        type: 'FeatureCollection',
-        features: geojson.features.map(function (feature) {
-            return cloneFeature(feature);
-        })
-    };
+    const cloned = {type: 'FeatureCollection'};
+    // Add id & bbox first
+    if (geojson.id) cloned.id = geojson.id;
+    if (geojson.bbox) cloned.bbox = geojson.bbox;
+
+    // Custom Properties
+    Object.keys(geojson).forEach(function (key) {
+        if (['id', 'type', 'bbox', 'features'].indexOf(key) !== -1) return;
+        cloned[key] = geojson[key];
```

Comment:
Shouldn't have any impact, before these properties were being dropped (so it's better than before).

We can add some tests for this behavior.

It shouldn't have any impact on the issue you mentioned above.